### ISSUE TITLE
Provide a way to zoom the projected-vs-real water height diagram in the Analysis view

### ISSUE THREAD

**Issue by nowakowski** [ismop/eitwid#126]
This concerns the diagram currently displayed in the top right-hand corner of the view. Requested by users.

**Comment by mkasztelnik** [comment]
You can do this by selecting interested time interval:

![water_hight_zoom](https://cloud.githubusercontent.com/assets/1265430/17848863/26b3e29c-6856-11e6-9075-55bfe1c08835.png)

**Comment by nowakowski** [comment]
No, they want a full-screen display (similar to the charts in the main view).

**Comment by mkasztelnik** [comment]
This is a feature request, cannot  be done in short period of time.

**Comment by nowakowski** [comment]
I know. I never said it was urgent. It was a feature request so I created a ticket; doesn't mean you must necessarily implement it.
